README.md

```diff
@@ -579,6 +579,21 @@ You can also use a JSON file with dokka configuration:
  java -jar <dokka_cli.jar> <path_to_config.json>
  ```
 
+Using CLI runner to generate default documentation requires providing all dependencies manually on classpath.
```

Comment:
Let's put CLI and maybe even maven plugin readmes into the subfolders 🙄 
This is just not relevant to most users 🙄

Reply:
This PR is just clarification for future useres about using CLI, I think moving these parts to subfolders we can discuss

Reply:
You could still link from the main readme to runners/cli/README.md 🙄 
I think the current readme is just toooo big now 🤖